code/modules/research/xenobiology/crossbreeding/burning.dm

```diff
@@ -53,6 +55,7 @@ Burning extracts:
 
 /obj/item/slimecross/burning/purple
 	colour = "purple"
+	effect_desc = "A clump of invigorating gel is created after use, it has healing properties and makes you feel good."
```

Comment:
A third style? I think it'd be better if it was consistently "Creates a clump of invigorating gel".